templates/core/terraform/locals.tf

```
@@ -21,3 +21,10 @@ data "http" "myip" {
 locals {
   myip = var.public_deployment_ip_address != "" ? var.public_deployment_ip_address : chomp(data.http.myip[0].body)
 }
+
```

Can you merge these 2 locals?
```suggestion
locals {
  myip = var.public_deployment_ip_address != "" ? var.public_deployment_ip_address : chomp(data.http.myip[0].body)
  tre_core_tags = {
    tre_id              = var.tre_id
    tre_core_service_id = var.tre_id
}
}
```

---

> Can you merge these 2 locals?

sure, done